src/wallet/wallet2.h

```diff
@@ -449,44 +449,48 @@ namespace tools
 
     /*!
      * \brief  Generates a wallet or restores one.
-     * \param  wallet_        Name of wallet file
-     * \param  password       Password of wallet file
-     * \param  multisig_data  The multisig restore info and keys
+     * \param  wallet_              Name of wallet file
+     * \param  password             Password of wallet file
+     * \param  multisig_data        The multisig restore info and keys
+     * \param  create_address_file  Whether to create an address file
      */
     void generate(const std::string& wallet_, const epee::wipeable_string& password,
-      const std::string& multisig_data);
+      const std::string& multisig_data, bool create_address_file = false);
 
     /*!
      * \brief Generates a wallet or restores one.
-     * \param  wallet_        Name of wallet file
-     * \param  password       Password of wallet file
-     * \param  recovery_param If it is a restore, the recovery key
-     * \param  recover        Whether it is a restore
-     * \param  two_random     Whether it is a non-deterministic wallet
-     * \return                The secret key of the generated wallet
+     * \param  wallet_              Name of wallet file
+     * \param  password             Password of wallet file
+     * \param  recovery_param       If it is a restore, the recovery key
+     * \param  recover              Whether it is a restore
+     * \param  two_random           Whether it is a non-deterministic wallet
+     * \param  create_address_file  Whether to create an address file
+     * \return                      The secret key of the generated wallet
      */
     crypto::secret_key generate(const std::string& wallet, const epee::wipeable_string& password,
       const crypto::secret_key& recovery_param = crypto::secret_key(), bool recover = false,
-      bool two_random = false);
+      bool two_random = false, bool create_address_file = false);
     /*!
      * \brief Creates a wallet from a public address and a spend/view secret key pair.
-     * \param  wallet_        Name of wallet file
-     * \param  password       Password of wallet file
-     * \param  viewkey        view secret key
-     * \param  spendkey       spend secret key
+     * \param  wallet_              Name of wallet file
+     * \param  password             Password of wallet file
+     * \param  viewkey              view secret key
+     * \param  spendkey             spend secret key
```

Comment:
Looks like the `address` argument is also undocumented here (which is also the case for another overload below). Also, the ordering of spendkey and viewkey is reversed. These are minor issues, but maybe you can fix them as well while you're at it?

Reply:
I'll open a separate PR to fix those comments.

Reply:
Fixed with https://github.com/monero-project/monero/pull/3405